spec/models/document_observer_spec.rb

```diff
@@ -53,10 +53,9 @@
   it "calls the DocumentsMailer, when a new document has been added" do
     document = FactoryGirl.build(:document)
     # make sure, that we have actually someone to notify
-    allow(document).to receive(:recipients).and_return(user.mail)
+    allow(document).to receive(:recipients).and_return([user])
     # ... and notifies are actually sent out
-    allow(Notifier).to receive(:notify?).and_return(true)
-
+    Setting.notified_events = Setting.notified_events << 'document_added'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.